scripts/west_commands/boards.py

```diff
@@ -85,4 +82,12 @@ def do_run(self, args, _):
             if name_re is not None and not name_re.search(board.name):
                 continue
             log.inf(args.format.format(name=board.name, arch=board.arch,
-                                       dir=board.dir))
+                                       dir=board.dir, hwm=board.hwm))
+
+        myboards = list_boards.find_v2_boards(args)
+        for board in list_boards.find_v2_boards(args):
+            b = board['board']
+            if name_re is not None and not name_re.search(board.name):
+                continue
+            log.inf(args.format.format(name=b['name'], dir=b['folder'],
+                                       hwm=b['hwm']))
```

Comment:
Formatting code should be made common and kept in `list_boards.py` for consistency. So far, each format call is missing a few different fields.

Reply:
the reason why they are different is because the existing `west boards` per default prints `arch`, however in board v2 the arch is no longer present as it has become meaningless.
Keeping `arch` in v1 printing as there could be external tools calling `west boards` and relying on this field.

Board v1 will be deprecated once all boards have been ported, therefore the code will eventually go away.

Therefore prefer to keep this code as two different paths.